src/assets/sass/protocol/components/_newsletter-form.scss

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-newsletter {
+    margin: 0 auto $margin-lg;
+    max-width: $width-content-sm;
+    padding: $padding-lg 0;
+
+    input[type='email'],
+    select {
+        width: 100%;
+    }
+
+    .mzp-c-form-submit {
+        text-align: center;
+    }
+
+    .mzp-c-button {
```

Comment:
Would this be worth having as a button style atom itself, as opposed to something specific to the newsletter form?

Reply:
Quite possibly, but I think Vince/Jennifer are still working on some button styles so I don't know if this will stick.